src/core/Table/Table.tsx

```diff
@@ -31,6 +32,8 @@ import SvgSortDown from '@itwin/itwinui-icons-react/cjs/icons/SortDown';
 import SvgSortUp from '@itwin/itwinui-icons-react/cjs/icons/SortUp';
 import { getCellStyle } from './utils';
 import { TableRowMemoized } from './TableRowMemoized';
+import { IconButton } from '../Buttons';
+import { SvgChevronDown, SvgChevronRight } from '@itwin/itwinui-icons-react';
```

Comment:
Import SVGs like we are importing other SVGs (lines 31, 32). We import like that because other projects does not use tree-shaking and their build will end up with all posible icons.